gpt_index/indices/knowledge_graph/retrievers.py

```diff
@@ -175,24 +180,19 @@ def _retrieve(
         )
         sorted_chunk_indices = sorted_chunk_indices[: self.num_chunks_per_query]
         sorted_nodes = self._docstore.get_nodes(sorted_chunk_indices)
-        # filter sorted nodes
-        postprocess_info = {"similarity_tracker": similarity_tracker}
-        for node_processor in self._node_postprocessors:
-            sorted_nodes = node_processor.postprocess_nodes(
-                sorted_nodes, postprocess_info
-            )
 
         # TMP/TODO: also filter rel_texts as nodes until we figure out better
         # abstraction
-        rel_text_nodes = [Node(text=rel_text) for rel_text in rel_texts]
-        for node_processor in self._node_postprocessors:
-            rel_text_nodes = node_processor.postprocess_nodes(rel_text_nodes)
-        rel_texts = [node.get_text() for node in rel_text_nodes]
+        # TODO(suo): figure out what this does
```

Comment:
this just injects the KG triplets as an extra "Node" in the retrieval step.

e.g. retrieval step will retrieve text Nodes (based on matching query to entities in KG), but will also add an extra Node itself like the following:
```
(subject, predicate, object)
(subject2, predicate2, object2)
```

etc. etc.